.vib/matomo/cypress/cypress/integration/matomo_spec.js

```diff
@@ -0,0 +1,67 @@
+/// <reference types="cypress" />
+import { random } from './utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.get('a[href*="logout"]').first().click();
+  cy.contains('Sign in');
+});
+
+it('allows to create user', () => {
+  cy.login();
+  cy.visit('/?module=UsersManager&showadduser=1');
+  cy.fixture('users').then((user) => {
+    cy.get('#user_login').type(`${user.newUser.username}-${random}`);
+    cy.get('#user_password').type(
+      `${user.newUser.password}-${random}`
+    );
+    cy.get('#user_password').type(
+      `${user.newUser.password}-${random}`
+    );
+    cy.get('#user_email').type(`${random}_${user.newUser.email}`);
+  });
+  cy.get('input[value*="Create user"]').click();
+  cy.contains('changes have been saved');
+});
+
+it('allows to create a new website', () => {
+  cy.login();
+  cy.visit('/index.php?module=SitesManager&showaddsite=1');
+  // We need to use the title attribute because with the inner HTML we cannot differentiate
+  // between "website" and "intranet website"
+  cy.get('button[title*="A website"]').click();
+  cy.fixture('websites').then((site) => {
+    // The name input has no attribute "name" or "id"
+    cy.get('input[placeholder="Name"]').type(`${site.newSite.name} ${random}`, { force: true });
+    cy.get('textarea[name="urls"]').type(`${site.newSite.url}`, { force: true });
+  });
+  cy.get('input[type="submit"]').click();
+  cy.contains('Website created');
+});
+
+it('allows to use the API', () => {
```

Comment:
Could you please specify what this API does?
If one day someone needs to fix the test it would help if this title clearly points to the area in the application.